ocdaction/tasks/migrations/0004_created_initial_anxiety_score_model.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2017-02-03 20:41
+from __future__ import unicode_literals
```

Comment:
Thanks for all suggestions, most useful (and for flak8 - good to know!) all updates made including the renaming .. please review once more